gk/main.c

```diff
@@ -733,10 +1187,27 @@ gk_stage1(void *arg)
 	return 0;
 }
 
+static void
+init_tunnel_mac(struct gk_config *gk_conf)
+{
+	unsigned int i;
+	struct gk_lpm *ltbl = &gk_conf->lpm_tbl;
+	for (i = 0; i < ltbl->num_policies; i++) {
+		/*
+		 * XXX The nexthop MAC addresses must
```

Comment:
This is a matter of semantic, but you meant a TODO tag, not an XXX tag.

From our roadmap:

All missing code must be tagged. Especially in the first patches, there will be chunks of missing code. All these chunks must have a description of what is missing preceded by one of the following tags: TODO or XXX. The tag TODO must be used when the missing code is needed for a functional build, whereas the tag XXX are for possible new features or improvements, and important suggestions.